admin/services/data-access/pupil-census-import.data.service.js

```diff
@@ -24,19 +23,17 @@ pupilCensusImportDataService.sqlBulkImport = async(pupilData, schools) => {
   } catch (error) {
     result.errorOutput = error
   }
+  con.release()
```

Comment:
how come you are releasing this explicitly?  are you managing the connection yourself now?

Reply:
Don't we want to release the connection back to the pool eventually?
I checked all the other use cases of `getConnection()` and `release()` is being called.

Reply:
forgot that you are requesting this yourself :)